icu4c/source/i18n/plurrule.cpp

```diff
@@ -1732,6 +1752,23 @@ int32_t FixedDecimal::getVisibleFractionDigitCount() const {
     return visibleDecimalDigitCount;
 }
 
+bool FixedDecimal::operator==(const FixedDecimal &other) const {
+    return source == other.source && visibleDecimalDigitCount == other.visibleDecimalDigitCount
+        && decimalDigits == other.decimalDigits && exponent == other.exponent;
+}
+
+UnicodeString FixedDecimal::toString() const {
```

Comment:
Where are you actually using `toString()`?

Reply:
I'm only using it for print debugging during the tests.  It's not necessary and can be removed once the test failures are resolved.

Reply:
`toString()` was also used in test error messages, which seems useful, so I kept it and fixed up the formatting and the compiler warning.  It's being called only in `PluralRulesTest::testGetSamples()`.